### DISCUSSION TITLE
Using values in variable groups in WikiUpdate

### DISCUSSION THREAD

**Discussion by rowbot-99** [rfennell/AzurePipelines#1332] [Q&A]
Is it possible to use azure devops variable groups when generating wiki updates. I am basically trying to add the URL of the build pipeline which I stored in a variable group.

**Comment by rfennell** [comment]
You can, see [this discussion](https://github.com/rfennell/AzurePipelines/discussions/983), but you can also get the pipeline URL from the `builddetails` object when generating release notes

Or have I misunderstood what you are trying to do?

**Comment by rowbot-99** [comment]
Thanks.. I am updating the wiki in the release pipeline, but I want to add
the URL of the build pipeline in the wiki. The URL is saved in a variable
group in azure DevOps. In my release-notes.md I added * **buildurl**:
$(URL) but it's obviously not returning values. I looked at the link you
sent but still not sure how I am meant to retrieve the values from the
variable group.

**Reply by rfennell** [reply]
Are you using my [Release Notes Task](https://marketplace.visualstudio.com/items?itemName=richardfennellBM.BM-VSTS-XplatGenerateReleaseNotes) to generate the wiki page? (for a full explanation of it usage see [video](https://www.youtube.com/watch?v=xaV3dFoQdV8))

If that is the case in the handlebars template the build URL is `{{builddetails.url}}`

This can be made into a Hyper link something like

```
[{{builddetails.buildNumber}}]({{builddetails.url}})
```
If this does not help, upload the YAML you are using so I can understand what you are trying

**Reply by rowbot-99** [reply]
Thanks, this has been resolved like this https://dev.azure.com/<org>/{{buildDetails.project.name}}/_build/results?buildId={{buildDetails.id}}&view=results
